### PR TITLE
fixing docstrings

### PR DESCRIPTION
**Pull request by mmatera**
more fixing in documentation and summary_texts

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/258"></a>

### CODE REVIEW

File: mathics/builtin/atomic/strings.py [hunk 1]
```diff
@@ -382,9 +384,16 @@ def to_python_encoding(encoding):
...
+    <dl>
+    <dt>'$CharacterEncodings'
+    <dd>stores the list of available character encodings.
+    </dl>
+    """
+
     name = "$CharacterEncodings"
     value = "{%s}" % ",".join(map(lambda s: '"%s"' % s, _encodings.keys()))
-
+    summary_text = "avaliable character encodings"
```
Comment (rocky) [inline mathics/builtin/atomic/strings.py:396]: avaliable -> available

File: mathics/builtin/atomic/strings.py [hunk 2]
```diff
@@ -450,6 +462,7 @@ class Alphabet(Builtin):
 
     """
 
+    summary_text = "lowercase letters in an alphabet"
```
Comment (rocky) [inline mathics/builtin/atomic/strings.py:465]: Would be nice to move this and the next one after messages.

File: mathics/builtin/inout.py [hunk 3]
```diff
@@ -2164,6 +2218,8 @@ class SympyForm(Builtin):
     = exp(2) + 3*E
     """
 
+    summary_text = "translate expressions to sympy"
```
Comment (rocky) [inline mathics/builtin/inout.py:2221]: BTW the person who worked on the project referred to the project as SymPy.

**Review by rocky — APPROVED** [review]
LGTM - but see comments.

**Comment by mmatera** [comment]
> Why do we need value at all here? Is it anywhere except in that rule. If so, it would be nice to explain what it use is.

I do not know, but changing it would be a detour from the goal of this PR. Let's merge and iterate. In next PRs I am going to continue sorting the properties

**Pull request merged**